spec/classes/server_spec.rb

```diff
@@ -1,177 +1,199 @@
 require 'spec_helper'
 
 describe 'mongodb::server' do
-  let :facts do
+  # Default facts
+  let(:default_facts) do
```

Comment:
this block shouldnt be needed. We already mock this fact here: https://github.com/voxpupuli/puppet-mongodb/blob/master/spec/default_module_facts.yml

Reply:
Ah, that's a useful method to know, as we have a number of occurrences of this on internal module tests...